kpops/components/base_components/pipeline_component.py

```diff
@@ -26,6 +27,11 @@ class PipelineComponent(BaseDefaultsComponent):
     from_: FromSection | None = Field(default=None, alias="from")
     app: object | None = None
     to: ToSection | None = None
+    prefix: str = Field(
+        default="${pipeline_name}-",
+        env=f"{ENV_PREFIX}PIPELINE_PREFIX",
```

Comment:
No env variable needed

Reply:
so, we only put the default as `${pipeline_name}-` ?

Reply:
Yes